.github/workflows/sync_branches.yml

```diff
@@ -1,9 +1,9 @@
 name: Sync Branches
 on:
   push:
-    branches:
-      - "2.5"
-      - "2.6"
+    # branches:
+    #   - "2.5"
+    #   - "2.6"
```

Comment:
why commented?

Reply:
I can't remember but I believe this was for testing. I have brought them back in!